sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/TextAnalyticsClientImpl.java

```diff
@@ -100,229 +102,197 @@ public TextAnalyticsClientImpl(HttpPipeline httpPipeline) {
+        Mono<SimpleResponse<SentimentResponse>> sentiment(@HostParam("Endpoint") String endpoint, @QueryParam("model-version") String modelVersion, @QueryParam("showStats") Boolean showStats, @BodyParam("application/json") MultiLanguageBatchInput input, Context context);
     }
 
     /**
-     * Named Entity Recognition
      * The API returns a list of general named entities in a given document. For the list of supported entity types, check &lt;a href="https://aka.ms/taner"&gt;Supported Entity Types in Text Analytics API&lt;/a&gt;. See the &lt;a href="https://aka.ms/talangs"&gt;Supported languages in Text Analytics API&lt;/a&gt; for the list of enabled languages.
-     *
-     * @param input Collection of documents to analyze.
-     * @param context The context to associate with this operation.
+     * 
+     * @param input Contains a set of input documents to be analyzed by the service.
+     * @param modelVersion (Optional) This value indicates which model will be used for scoring. If a model-version is not specified, the API should default to the latest, non-preview version.
+     * @param showStats (Optional) if set to true, response will contain input and document level statistics.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @return a Mono which performs the network request upon subscription.
+     * @throws TextAnalyticsErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<SimpleResponse<EntitiesResult>> entitiesRecognitionGeneralWithRestResponseAsync(MultiLanguageBatchInput input, Context context) {
-        final String modelVersion = null;
-        final Boolean showStats = null;
-        return service.entitiesRecognitionGeneral(this.getEndpoint(), modelVersion, showStats, input, context);
+    public Mono<SimpleResponse<EntitiesResult>> entitiesRecognitionGeneralWithResponseAsync(MultiLanguageBatchInput input, String modelVersion, Boolean showStats) {
+        return FluxUtil.withContext(context -> service.entitiesRecognitionGeneral(this.getEndpoint(), modelVersion, showStats, input, context));
     }
 
     /**
-     * Named Entity Recognition
      * The API returns a list of general named entities in a given document. For the list of supported entity types, check &lt;a href="https://aka.ms/taner"&gt;Supported Entity Types in Text Analytics API&lt;/a&gt;. See the &lt;a href="https://aka.ms/talangs"&gt;Supported languages in Text Analytics API&lt;/a&gt; for the list of enabled languages.
-     *
-     * @param input Collection of documents to analyze.
+     * 
+     * @param input Contains a set of input documents to be analyzed by the service.
+     * @param context The context to associate with this operation.
```

Comment:
What is this context change?

Reply:
This is the generated code, regenerated with using autorest with new swagger. 
There will be no manual update on this file.